### ISSUE TITLE
new report there is no focus indicator

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-web#811]
## Describe the bug
chevron to expand/collapse in report should have focus indicator

**To Reproduce**

1. Go to any website
2. run automated checks
3. download report and add a comment with line breaks and spaces
4. open report

## Expected behavior
there is visible focus indicator

## Extension (please complete the following information)

- OS: [e.g. Windows/Mac] Windows 
- Version [e.g. 1.140.1] 2.5.1
- Browser Version [e.g Chrome 71.0] 74
- Target Page [e.g www.bing.com] any

## Are you willing to submit a PR?

If asked, will you be willing to submit a PR to fix this bug? No

## Did you search for similar existing issues?

Did you search for similar issues in our github issues or on [stackoverflow](https://stackoverflow.com/questions/tagged/accessibility-insights) Yes

## Additional context

Found during validation

**Comment by smoralesd** [comment]
fixed on #810
